opentelemetry-sdk/setup.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import setuptools
+
+base_dir = os.path.dirname(__file__)
+
+package_info = {}
+with open(os.path.join(base_dir, "src", "opentelemetry", "sdk", "version.py")) as f:
+    exec(f.read(), package_info)
```

Comment:
Why use `package_info` instead of read/exec-ing right into `globals`?

Reply:
To prevent bad thing inside the `version.py` (later we might add more stuff such like SDK vendor info and rename the file to package_info.py) polluting globals.